src/store/login/action.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  UPDATE_ACCOUNT,
+  UPDATE_PASSWORD,
+  UPDATE_TOKEN,
+  UPDATE_LOADING,
+  FETCH_DATA,
+  UpdateAccountAction,
+  UpdatePasswordAction,
+  UpdateTokenAction,
+  UpdateLoadingAction,
+  FetchDataAction,
+} from './types';
+
+export function updateAccount(params: string): UpdateAccountAction {
```

Comment:
I'd propose you to use [typesafe-actions](https://github.com/piotrwitek/typesafe-actions) for creating actions/reducers. It provides awesome type definitions for all this things